.github/workflows/lp-to-gh-issues.yml

```diff
@@ -0,0 +1,19 @@
+name: LP to GH issue migator
+on:
+  schedule:
+    - cron:  '0 22 * * *'
+
+jobs:
+  lp-to-gh-issues:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
```

Comment:
Surprised that you need this = doesn't `ubuntu-latest` come with python 3.8 already? We've never had to set it up manually in our other actions - e.g. [here](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/.github/workflows/pr.yaml#L92)

Reply:
I think this is to ping the version of python used. Got it from here: https://docs.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions

Reply:
In that case they do it to run with all sorts of python versions. I really don't think it's necessary. I know this is merged, but I'd appreciate it if you would remove this if you get a chance.